### DISCUSSION TITLE
Is there a way to get Revisions?

### DISCUSSION THREAD

**Discussion by urbgimtam** [Intevel/nuxt-directus#71] [Q&A] [answered]
Is there a way to get Revisions?

At the moment, I need to fetch them directly.

**Comment by Intevel** [comment, marked as answer]
Currently not, but I can implement this if you want :)

**Reply by urbgimtam** [reply]
Thanks. I've managed to go directly by rest, but I think it would be a great feature.

**Reply by Intevel** [reply]
https://github.com/Intevel/nuxt-directus/releases/tag/v3.1.0
